components/GridView.vue

```diff
@@ -29,14 +28,25 @@ const gridOptions = reactive<GridProperties>({
   isResponsive: true,
 })
 const layout = ref<GridLayoutItem[]>([])
+
+const responsiveLayouts = reactive({
+  xxs: [],
+  xs: [],
+  sm: [],
+  md: [],
+  lg: [],
+} as Record<Breakpoint, GridLayoutItem[]>)
```

Comment:
Does it makes sense to define so many layouts if we use only 1 and 2 column layout?

Reply:
Removed the commit that addressed responsiveness since it was not working correctly. Will address that in a follow up PR